### DISCUSSION TITLE
p.247 リスト20.27のbad_req.json.goldenについて

### DISCUSSION THREAD

**Discussion by kdnakt** [budougumi0617/go_todo_app#49] [Ideas]
```
{
  "user_name": "budougumi0617",
  "password": "test"
}
```

となっていますが、bad_req_rsp.json.goldenの内容からすると、passwordフィールドがなくてログインに失敗するテストのようなので、以下のようにpasswordフィールドを削除する必要がありそうです。

```
{
  "user_name": "budougumi0617"
}
```

**Comment by budougumi0617** [comment]
ご指摘ありがとうございます。わかりにくいのですが `bad_req.json.golden` に含まれているのはよく見ると `passwOrd` フィールドではなくて、 `passwArd` フィールドだったりします。なので、このリクエストJSONと `password`フィールドがないという期待レスポンスJSONの組み合わせで正しいはずです。
「タイポしていてもvalidationがあれば気づけますね」ということを示したかった参考例なのでわかりづらくて申し訳ないです 🙇 

https://github.com/budougumi0617/go_todo_app/blob/v1.0.6/handler/testdata/login/bad_req.json.golden#L3
![image](https://user-images.githubusercontent.com/1883265/189701993-e04ba4cd-2fde-4c3e-9be7-1e5dc73b72c8.png)

**Reply by kdnakt** [reply]
なるほど、よく読んでいなくてすみませんでした 🙇
